src/blocks/icon/index.php

```diff
@@ -77,12 +77,15 @@ function coblocks_render_icon_block( $attrs ) {
 	$custom_icon_path = is_readable( "$custom_icon_path$icon_style.svg" ) ? "$custom_icon_path$icon_style.svg" : $custom_icon_path . '.svg';
 	$icon_path        = is_readable( "$icon_path$icon_style.svg" ) ? "$icon_path$icon_style.svg" : $icon_path . '.svg';
 
+	// Needed for the call to $wp_filesystem->get_contents(...) .
+	require_once ABSPATH . 'wp-admin/includes/file.php';
```

Comment:
@olafleur-godaddy I would use include or simply require as this will throw an error if another plugin already included the file. Maybe also add a check if the wp filesystem class is already in the scope.